app/DoctrineMigrations/Version191.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version191 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE INDEX report_status_cached_idx ON report (report_status_cached)');
```

Comment:
we already have a report.statusCached. what is the difference between the two?

Reply:
report.statusCached holds an array with section statuses, the first solution I implemented, that turned out not to be enough. I had to store the final report status  "report_status_cached" and use an index on them in order to avoid fetching all the reports to calculate the ones to display in the page, and the counts for the tabs